hoomd/simulation.py

```diff
@@ -213,7 +214,10 @@ def create_state_from_snapshot(self, snapshot):
     @property
     def state(self):
         """hoomd.State: The current simulation state."""
-        return self._state
+        if self._state is None:
+            raise DataAccessError("state")
+        else:
+            return self._state
```

Comment:
```suggestion
        return self._state
```
I think a `None` return value here makes sense. The state has not yet been initialized.